press/press/doctype/team/suspend_sites.py

```diff
@@ -8,83 +8,62 @@
 
 Defaulters are identified based on the following conditions:
 - Is not a free account
-- Is not an ERPNext Partner
-- Card not added
-- Does not have enough credit balance
+- Is not a Legacy Partner account with payment mode as Partner Credits
+- Has at least one unpaid invoice
+- Has an active site
 
-The `execute` method is the main method which is run by the scheduler daily.
+The `execute` method is the main method which is run by the scheduler on every 10th day of the month.
 """
 
 
 import frappe
-from frappe.utils.data import flt
 
 
 def execute():
-	teams_with_total_usage = get_teams_with_total_usage()
+	teams_with_unpaid_invoices = get_teams_with_unpaid_invoices()
 
-	for d in teams_with_total_usage:
-		total_usage = d.total_usage
+	for d in teams_with_unpaid_invoices:
 		team = frappe.get_doc("Team", d.team)
 
-		if team.free_account or not total_usage or team.get_balance() > 0:
+		if team.free_account or team.payment_mode == "Partner Credits":
```

Comment:
We are not suspending for payment_mode = "Partner Credits"
@shadrak98 @michellealva

Reply:
Currently not, as there are few partners who are on legacy contract. This will be removed later